docs/guide/release.md

```diff
@@ -0,0 +1,42 @@
+# Release guidelines
+
+
+odfe-cli versions are expressed as **x.y.z**, where **x** is the major version, **y** is the minor version, and **z** is the patch version, following [Semantic Versioning](https://semver.org/) terminology. 
+
+**The major, minor & patch version is related to odfe-cli compatibility, instead of odfe’s compatibility.**
+
+
+**Q: What goes into new patch version release train?**
+
+* Known bugs that prevent users from executing tasks. 
+    * eg: Failed to delete non-running detectors by delete command, failed to handle error.
+* Security vulnerabilities. 
+    * eg: leaking user credentials, insecure interaction with components.
+* cli’s dependency is updated with new patch version which fixes bugs and security issues that affects your feature. ( Treat your dependent library as your library)
+    * eg: cobra released a new version which fixes some of security vulnerabilities like [#1259](https://github.com/spf13/cobra/pull/1259)
+
+Any committer, who fixed issues, **related to above use case**, should notify admin to initiate new patch release. There is no SLA for patch releases, since this is triggered based on severity of the issue.
```

Comment:
So if a community contributer makes a patch, who should they notify?

Reply:
We will be publishing plugin owners github/email id . Hence, when a community member contributes a patch, mostly to a plugin, the plugin owner who merged the code should take a call. What do you think?

Reply:
That makes sense